### PR TITLE
mgr/dashboard: Export rgw multisite status via the API

### PR DESCRIPTION
**Pull request by aaSharma14**
Export rgw multisite status via the API

Fixes: https://tracker.ceph.com/issues/64081

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/controllers/rgw.py [hunk 1]
```diff
@@ -110,6 +110,9 @@ def migrate(self, daemon_name=None, realm_name=None, zonegroup_name=None, zone_n
                                                          secret_key)
         return result
 
+
+@APIRouter('rgw/multisite', Scope.RGW)
```
Comment (nizamial09) [inline src/pybind/mgr/dashboard/controllers/rgw.py:114]:
```suggestion
@APIRouter('rgw/multisite', Scope.RGW)
@APIDoc("RGW Multisite Management API", "RgwMultisite")
```

File: src/pybind/mgr/dashboard/controllers/rgw.py [hunk 2]
```diff
@@ -110,6 +110,9 @@ def migrate(self, daemon_name=None, realm_name=None, zonegroup_name=None, zone_n
...
+class RgwMultisiteSyncStatus(RESTController):
     @RESTController.Collection(method='GET', path='/sync_status')
```
Comment (nizamial09) [inline src/pybind/mgr/dashboard/controllers/rgw.py:116]:
```suggestion
    @RESTController.Collection(method='GET', path='/sync_status')
    @EndpointDoc("Get the multisite sync status")
```

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.ts [hunk 3]
```diff
@@ -48,9 +48,10 @@ export class RgwOverviewDashboardComponent implements OnInit, OnDestroy {
   BucketSub: Subscription;
   queriesResults: { [key: string]: [] } = {
     RGW_REQUEST_PER_SECOND: [],
-    BANDWIDTH: [],
     AVG_GET_LATENCY: [],
-    AVG_PUT_LATENCY: []
+    AVG_PUT_LATENCY: [],
+    GET_BANDWIDTH: [],
+    PUT_BANDWIDTH: [],
```
Comment (nizamial09) [inline src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.ts:54]: why this change?

**Comment by aaSharma14** [comment]
jenkins test make check

**Review by avanthakkar — APPROVED** [review]
